New/src/content/en/events/_index.yaml

```diff
@@ -0,0 +1,69 @@
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+landing_page:
+  custom_css_path: /web/events/events.css
+  header:
+    description: Attend A Developer Event And Get Hacking
+
+  rows:
+
+  - heading: Find Events Near You
+    classname: find-events
+    items:
+    - description: >
+        <p>
+          Events are run by local Google Developer Groups or other
+          developer meetups. Each event is a little different, but generally, an
+          event can include a tech talk, series of code labs, and time for
+          personal coding. Find one in your area or start your own!
+        </p>
+
+        <p>
+          If you're not able to find a nearby event don't worry! All the
+          materials are available so you can organize your own event or
+          work through the content at home. <a href="#featured-events">See the
+          featured events section for more details</a>.
+        </p>
+
+    - custom_html: >
+        <div class="doc-iframe-container">
+          {% framebox height="500px" %}
+            <link rel="stylesheet" href="/web/events/embeds/event-map.css">
+            <div id="map" data-tags="pwaroadshow,polymer"></div>
```

Comment:
I don't think there are any events actually using the tag yet
Here's the results for polymer
http://web-events-feed.appspot.com/feeds?tags=polymer

Here's the results for pwaroadshow
http://web-events-feed.appspot.com/feeds?tags=pwaroadshow